modules/fedimint-ln-client/src/lib.rs

```diff
@@ -1457,6 +1457,10 @@ impl LightningClientModule {
             }
         }))
     }
+
+    pub async fn get_unspent_incoming_contract_hashes(&self) -> anyhow::Result<Vec<sha256::Hash>> {
```

Comment:
I'm unsure do we need to increment Api version requirement for client module now, since we are adding a method that server needs to have to support this version of a client.

On the server side we might have already increased the api version advertised for some recovery-related stuff that was unreleased, but we didn't need to do it for the client, as it was able to auto-detect that's the case and hadle it gracefully.

I guess if you dont not want to require newer server, you'd need to make this `get_unspent_incoming_contract_hashes_opt(..) -> Result<Option<...>>` and return `Ok(None)` if server doesn't support it?

See https://github.com/fedimint/fedimint/blob/02abf76eb111682b5e8d38ad7288a716934fe76a/fedimint-client/src/module/init/recovery.rs#L268